dev/tests/drag.tsx

```diff
@@ -2,6 +2,15 @@ import { motion } from "@framer"
 import * as React from "react"
 
 // It's important for this test to only trigger a single rerender while dragging (in response to onDragStart) of draggable component.
```

Comment:
Should this comment go below the helper function? Or maybe the helper function should move to the bottom of the file?